platooning_strategic/src/platoon_manager.cpp

```diff
@@ -133,7 +134,7 @@ namespace platoon_strategic
 
     int PlatoonManager::getTotalPlatooningSize() const{
         if(isFollower) {
-            return platoonSize;
+            return std::max((int)(platoon.size() + 1), platoonSize);
```

Comment:
I'm not sure I follow what this is accomplishing, platoonSize here is the number of vehicles in the platoon? And platoon is the list of those vehicles? I'm not sure i understand how they would be a different number.

More minor: Is the (int) cast needed simply because std::max can't work on a size_t (from platoon.size()) and an int?

Reply:
platoonSize is the size of the full platoon. However, platoon vector includes only the vehicles in front of the host vehicle. Therefore, unless the host is the last follower, the platoon.size() will not show the correct platoon size.